java/src/main/java/ai/rapids/cudf/Rmm.java

```diff
@@ -88,68 +88,37 @@ public static LogConf logToStderr() {
    *                       {@link RmmAllocationMode#ARENA},
    *                       {@link RmmAllocationMode#CUDA_ASYNC} and
    *                       {@link RmmAllocationMode#CUDA_MANAGED_MEMORY}
-   * @param enableLogging  Enable logging memory manager events
+   * @param logConf        How to do logging or null if you don't want to
    * @param poolSize       The initial pool size in bytes
    * @throws IllegalStateException if RMM has already been initialized
    */
-  public static void initialize(int allocationMode, boolean enableLogging, long poolSize)
```

Comment:
Curious why this API was removed as part of this change?  It wasn't marked deprecated and a max pool size parameter isn't involved.

Reply:
Just trying to simplify the api since this is not used in the plugin.